### DISCUSSION TITLE
Setting two separate effects to ligand and protein

### DISCUSSION THREAD

**Discussion by rez3vil** [schrodinger/pymol-open-source#318] [Q&A]
Hi,

I want to make an image with following settings

to protein > cartoon_transparency, 0.9 / ray_trace_mode, 1 / ambient, 0.5
to ligand > default values

But whenever I select just protein and apply effects, its applied to ligand as well. 

Is there a way to achieve this? I mean having two separate effects given in the same file. 

Thanks.

**Comment by JarrettSJohnson** [comment]
`ambient` and `ray_trace_mode` are both global settings. They'd need to be drawn separately and composited.
